sdk/app_cpu1/common/drv/pwm.c

```diff
@@ -110,7 +144,7 @@ void pwm_set_all_rst(uint8_t rst)
     value |= (uint32_t) rst;
 
     // Offset 27 is rst output reg
```

Comment:
No need for this comment now, duplication of source of truth

Reply:
I removed the other similar comments.